api/queue/work/model_service_deployment.go

```diff
@@ -23,7 +24,7 @@ var deploymentCounter = prometheus.NewCounterVec(
 		Namespace: "merlin_api",
 		Help:      "Number of deployment",
 	},
-	[]string{"project", "model", "status"},
+	[]string{"project", "model", "status", "redeploy"},
```

Comment:
Would be good to mention changes like this (also more details about the new deployment history view) in the PR description, for future reference.